tests/generate_example_data.py

```
@@ -0,0 +1,361 @@
+    df = pd.DataFrame({
+        'dw_ek_kontakt': dw_eks_kontakt,
+        'diagnosekode': generate_medical_code(n_total, prefix="D"),
+        'diagnosetype': [random.choices(['A', 'B', '+'], weights=[15,80,5])[0] for _ in range(n_total)],
+        'senere_afkraeftet': [random.choices(['Nej', 'Ja'], weights = [99,1])[0] for _ in range(n_total)],
+        'lpindberetningssystem': ['LPR3' for _ in range(n_total)],
+    })
+    os.makedirs(save_dir, exist_ok=True)
+    df.to_csv(f'{save_dir}/concept.register_diagnosis.csv', index=False)    
+
+def generate_forloeb(mapping_merged):
+    filtered_pids = mapping_merged[(mapping_merged['forloeb'] == True) & (mapping_merged['CPR_hash'].isna())]['PID'].values
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if mapping_merged['forloeb']:` for truth checks

Replace with `mapping_merged['forloeb']`

(E712)